tests/unit/stores/UserStore.spec.js

```diff
@@ -0,0 +1,26 @@
+import { setActivePinia, createPinia } from 'pinia';
+import { useUserStore } from 'src/stores/UserStore';
+
+describe('User Store', () => {
+  beforeEach(() => {
+    // creates a fresh pinia and makes it active
+    // so it's automatically picked up by any useStore() call
+    // without having to pass it to it: `useStore(pinia)`
+    setActivePinia(createPinia());
+  });
+
+  it('should test variables in the store', () => {
```

Comment:
missing test username empty and firstname not

Reply:
And separate test into multiple it like
describe('Test getter: isEmpty')
with it(should return true on empty username and firstname) etc etc

Reply:
Done